src/index.html

```diff
@@ -14,8 +14,57 @@
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto&display=swap">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="mainCard" data-qa="card">
+      <div class="mainCard__image image">
+        <img class="photo"
+        src="images/imac.jpeg" alt="mac">
+      </div>
+
+      <div class="mainCard__description">
```

Comment:
for everything you do not required `div`, only for `stars` block you will need it